### DISCUSSION TITLE
Notifications disabled

### DISCUSSION THREAD

**Discussion by pekoloko-cmd** [Atlas-OS/Atlas#1172] [Help & Support] [answered]
Hi i downloaded atlas-os but i need notifications enabled so i went into the atlas folder to run the script but when i restarted my pc the notifications was still disabled can you help?

**Comment by he3als** [comment, marked as answer]
This is a known 0.4 issue and will be fixed in the next release.

See https://github.com/orgs/Atlas-OS/projects/7/views/5 for all known v0.4 issue workarounds, which include the notifications page in Settings not re-enabling.
